### PR TITLE
app: stop pinging blocklisted users

### PR DESCRIPTION
**Pull request by Erikvv**
I am on the blocklisted users but I'm still getting pings. This
hopefully fixes that. Not tested.

### CODE REVIEW

File: app/core.go [hunk 1]
```diff
@@ -1775,6 +1775,9 @@ func (a *App) formatChangeInfoLink(ch *gerrit.ChangeInfo) string {
 
 func (a *App) prepareUserLink(ctx context.Context, account *events.Account) string {
 	chatID := a.lookupGerritUser(ctx, account)
+	if a.isBlocklisted(ctx, chatID) {
+		return chatID
```
Comment (thepaul) [inline app/core.go:1779]: `chatID` will be a string like `"U037Q3NGS9B"`, which will be unhelpful to the average reader. We should return `account.DisplayName()` instead.

**Review by thepaul — CHANGES_REQUESTED** [review]
This changes the semantics of the `blocklist-ids` config item, so it would be nice to update the comment on that as well (line 79 should say something like "comma-separated list of chat IDs of users to avoid messaging *or pinging*").

**Comment by Erikvv** [comment]
Maybe you can explain better what this tool can help me with so maybe I don't have to go for a full mute but constomize it otherwise.

I am on the blocklist but I remember I mainly asked it to stop these 5 PM pings which I don't want because I'm acutely aware of all the open changes anyways. 

I had hoped to rely on notifications for github and e-mail filters for gerrit. But the github notifications are just too much because I get everything from all teams, and protonmail's e-mail filters are too limited. So I am just overwhelmed and it has all become pretty useless...

**Comment by Erikvv** [comment]
We aren't using this tool currently so I'm shelving this.

**Pull request closed without merging**